install/installer/traf_config_setup

```diff
@@ -407,61 +415,270 @@ else
 fi
 
 #==============================================
-#Hadoop user name
+#Enter Hadoop Type
 
-echo -n "Enter Hadoop admin username, default is [$ADMIN]: "
-read answer
+echo -n "Specify the version of Hadoop installed (1: Cloudera, 2: Hortonworks, 3: Other): "
+read answer1
+  if [[ -z "$answer1" ]]; then
+     echo "***ERROR: User must specify the version of Hadoop being used."
+     exit -1
+  fi
+  case "$answer1" in
+    1) HADOOP_TYPE="cloudera"
+       echo "export HADOOP_TYPE=\"$HADOOP_TYPE\"" >> $LOCAL_TRAF_CONFIG
+       ;;
+    2) HADOOP_TYPE="hortonworks"
+       echo "export HADOOP_TYPE=\"$HADOOP_TYPE\"" >> $LOCAL_TRAF_CONFIG
+       ;;
+    3) HADOOP_TYPE="apache"
+       echo "export HADOOP_TYPE=\"$HADOOP_TYPE\"" >> $LOCAL_TRAF_CONFIG
+  esac
 
-if [ -z $answer ]; then
-   echo "export ADMIN=\"$ADMIN\"" >> $LOCAL_TRAF_CONFIG
-   username=$ADMIN
-else
-   echo "export ADMIN=\"$answer\"" >> $LOCAL_TRAF_CONFIG
-   username=$answer
-fi
+export $HADOOP_TYPE
 #==============================================
-#Hadoop Password
+if [[ "$HADOOP_TYPE" == "apache" ]]; then
+   echo -n "Enter Hadoop installed full path (example: '/opt/hadoop-2.6.0'), default is [$HADOOP_PREFIX]: "
+   read answer
 
-echo -n "Enter Hadoop admin password, default is [$PASSWORD]: "
-read answer
+   if [ -z  $answer ]; then
+      if [[ -z $HADOOP_PREFIX ]]; then
+         echo "***ERROR: Must enter Hadoop installed path"
+         exit -1
+      fi
+   else
+      if [[ -e $answer ]]; then
+         HADOOP_PREFIX=$answer
+      else
+         echo "***ERROR: Hadoop installed path doesn't exist"
+         exit -1
+      fi
+   fi
+   echo "export HADOOP_PREFIX=\"$HADOOP_PREFIX\"" >> $LOCAL_TRAF_CONFIG
+
+   #Hbase path 
+   echo -n "Enter HBase installed full path (example: '/opt/hbase-1.2.0'), default is [$HBASE_HOME]: "
+   read answer
+
+   if [ -z  $answer ]; then
+      if [[ -z $HBASE_HOME ]]; then
+         echo "***ERROR: Must enter HBase installed path"
+         exit -1
+      fi
+   else
+      if [[ -e $answer ]]; then
+         HBASE_HOME=$answer
+      else
+         echo "***ERROR: HBase installed path doesn't exist"
+         exit -1
+      fi
+   fi
+   echo "export HBASE_HOME=\"$HBASE_HOME\"" >> $LOCAL_TRAF_CONFIG
+   
+   #Zookeeper path
+   echo -n "Enter Zookeeper installed full path (example: '/opt/zoo'), default is [$ZOO_HOME]: "
+   read answer0
+
```

Comment:
I think I had a reason at the time but I don't think it needs to be this now. 

Thanks. I will change it.